hdmi/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.0'
```

Comment:
[Versions](http://effbot.org/pyfaq/how-does-the-python-version-numbering-scheme-work.htm) are always interesting and fun.  Review the link for a brief description of the version scheme used in Python.  During development, **pre** the first release, I like to increment the "bug" number and target the first release to be `0.1.0`.  At this point I would version it as `0.0.1`.